src/js/views/programs/list/programs-list.scss

```diff
@@ -0,0 +1,7 @@
+.program--on {
+  color: color(green);
+}
+
+.program--words {
+  margin-left: 6px;
```

Comment:
I'm nearly sure we should only be using 4px increments, so 4 or 8.  That said there's a little more styling to do, so I wouldn't worry too much about it.. can get caught on a second pass.